platform-stack/docker-compose.yml

```diff
@@ -56,6 +56,28 @@ services:
     networks:
       - internal
 
+  deployment:
+    image: miragon/miranum-deployment-service:latest
+    container_name: miranum-deployment
+    ports:
+      - "8092:8080"
+    restart: unless-stopped
+    env_file:
+      - .env
+    environment:
+      MIRANUM_DEPLOYMENT_SERVER_BPMN_TEST_URL: http://host.docker.internal
```

Comment:
I like that you chose to communicate with the service directly instead of using the nginx endpoint. We should use that way in kubernetes as well!